fixtures/docstring-proc-macro/src/docstring-proc-macro.udl

```diff
@@ -0,0 +1,2 @@
+/// <docstring-namespace>
```

Comment:
side-note: fwiw, when I implemented this I didn't use the UDL at all and only then realized that of course we can't get the crate's top-level docs as the namespace docs. But I guess that's fine.